src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -1264,11 +1267,12 @@ private IAddressOfOperation CreateBoundAddressOfOperatorOperation(BoundAddressOf
 
         private IInstanceReferenceOperation CreateBoundImplicitReceiverOperation(BoundImplicitReceiver boundImplicitReceiver)
         {
+            InstanceReferenceKind referenceKind = InstanceReferenceKind.ImplicitReceiver;
```

Comment:
>InstanceReferenceKind.ImplicitReceiver [](start = 50, length = 38)

Could you please point me to IOperation tests that go through this code path?

Reply:
Some of the tests I found:
* ObjectCreationWithArrayInitializer
* ObjectCreationWithInvalidMemberInitializer
* ObjectCreationWithInvalidInitializer
* DeconstructDelegate2
* ObjectInitializerTest_IncompleteComplexElementInitializerExpression
* CS0246_ERR_SingleTypeNameNotFound
* CS0117ERR_NoSuchMember
* AddPropertyUsedLikeMethod

---
In reply to: [180823954](https://github.com/dotnet/roslyn/pull/26055#discussion_r180823954) [](ancestors = 180823954)